### ISSUE TITLE
Google Street View Sphere

### ISSUE THREAD

**Issue by LorenDorez** [google/marzipano#263]
When i try to display an image captured by google street view app it just loads a black screen. Below is the pic details and how i renter it.

  GPano:CroppedAreaImageHeightPixels: 3092
          GPano:CroppedAreaImageWidthPixels: 16896
          GPano:CroppedAreaLeftPixels: 0
          GPano:CroppedAreaTopPixels: 2588
          GPano:FirstPhotoDate: 2019-08-15T18:36:38.19Z
          GPano:FullPanoHeightPixels: 8448
          GPano:FullPanoWidthPixels: 16896
          GPano:LastPhotoDate: 2019-08-15T18:37:10.975Z
          GPano:PoseHeadingDegrees: 76.0
          GPano:ProjectionType: equirectangular
          GPano:SourcePhotosCount: 12

```
 // Create viewer.
            var viewer = new Marzipano.Viewer(document.getElementById('pano'));

            // Create source.
            var source = Marzipano.ImageUrlSource.fromString(
                "https://localhost:5001/images/captured/Google_StreetView_Pano.jpg"
            );

            // Create geometry.
            var geometry = new Marzipano.EquirectGeometry([{ width: 4000 }]);

            // Create view.
            var limiter = Marzipano.RectilinearView.limit.traditional(1024, 100 * Math.PI / 180);
            var view = new Marzipano.RectilinearView({ yaw: Math.PI }, limiter);


            // Create scene.
            var scene = viewer.createScene({
                source: source,
                geometry: geometry,
                view: view,
                pinFirstLevel: true
            });

            // Display scene.
            scene.switchTo();
```

**Comment by tjgq** [comment]
I don't see anything obviously wrong with your code.

Usually the black screen occurs when you're visiting the page through a `file://` URL (i.e., loading the index.html file directly in your browser) instead of running a webserver. Is this the case?

If not, are there any error messages in the developer console?

**Comment by LorenDorez** [comment]
No, im running through IIS. I can load other images just fine, thats the only one not working. No errors in the console either. Ill PM you the link to the photo to test.
